sphinx/setup_command.py

```diff
@@ -144,15 +145,19 @@ def finalize_options(self):
             self.config_dir = self.source_dir
         self.config_dir = abspath(self.config_dir)
 
+        self.ensure_string_list('builder')
```

Comment:
I'm sorry, but could you add `# type: ignore` label?
It seems mypy does not know `ensure_string_list()` unfortunately. 
https://travis-ci.org/sphinx-doc/sphinx/jobs/205824862#L593